ui/src/dashboards/components/TemplateControlBar.js

```diff
@@ -57,6 +57,44 @@ const TemplateControlBar = ({
 
 const {arrayOf, bool, func, shape, string} = PropTypes
 
+class ControlBarWrapper extends Component {
+  constructor(props) {
+    super(props)
+  }
+
+  componentDidMount() {
+    const {source, dashboard, updateTempVarValues} = this.props
+    updateTempVarValues(source, dashboard)
+  }
+
+  render() {
+    const {
+      isOpen,
+      dashboard,
+      onSelectTemplate,
+      onOpenTemplateManager,
+    } = this.props
+
+    return (
+      <TemplateControlBar
+        isOpen={isOpen}
+        templates={dashboard.templates}
+        onSelectTemplate={onSelectTemplate}
+        onOpenTemplateManager={onOpenTemplateManager}
+      />
+    )
+  }
+}
+
+ControlBarWrapper.propTypes = {
+  isOpen: bool,
+  source: shape().isRequired,
+  dashboard: shape().isRequired,
+  onSelectTemplate: func.isRequired,
+  updateTempVarValues: func.isRequired,
+  onOpenTemplateManager: func.isRequired,
+}
+
 TemplateControlBar.propTypes = {
```

Comment:
@jademcgough we moved the async action from `TemplateControlBar` which means we no longer need the wrapper component :)